examples/features_demo/README.md

```diff
@@ -2,12 +2,66 @@
 
 This examples demonstrates Yarner's most important features by creating a simple but complete [Rust](https://rust-lang.org) project.
```

Comment:
I guess the overlap between the default setup and this one is by now quite high? What do you think about having `examples/rust` and `examples/java` showing how to do a small Gradle project instead to highlight that Yarner is not married to Rust. (And of course, `examples/netlogo` if you feel adventurous...) Personally, I think the book is a better place to ensure feature coverage as I prefer examples to are easy to digest.

Reply:
Too bad that reverse mode is not possible with NetLogo, as it allows for comments only in the code part, but not in the rest. Using `.nls` files could be a solution, however.

Reply:
Also, I realize that there should be a way to define language settings while still having no block labels in the output, which is cuttently not possible.

Reply:
See #68 and #67

Reply:
Regarding the examples, do you mean to have a single example for each language, or more like having e.g. the two rust examples in a sub-folder `rust`?

Reply:
No, I meant a single example for each language which keeps it simple so it is easy to understand.